### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by SetiZ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+run `docker-compose`
+```
+docker-compose up -d
+```
+
+### api
+
+server will start on port `3000`.
+
+You then will be able to make a `POST` request to `localhost:3000` with plain text data and the response will generate a pdf for you.
```
Comment (jvasseur) [inline README.md:29]:
```suggestion
You then will be able to make a `POST` request to `localhost:3000` with HTML data and the response will generate a pdf for you.
```

**Review by jvasseur — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# html-t-pdf
```
Comment (jvasseur) [inline README.md:1]:
```suggestion
# html-to-pdf
```

**Review by jvasseur — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (SetiZ) [reply README.md:1]: rooh la boulette

File: README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# html-to-pdf
+this is a api that can be used to generate a pdf from a source file.
```
Comment (jvasseur) [inline README.md:2]:
```suggestion
this is an API that can be used to generate a PDF from a source file.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+run `docker-compose`
+```
+docker-compose up -d
+```
+
+### api
+
+server will start on port `3000`.
+
+You then will be able to make a `POST` request to `localhost:3000` with HTML data and the response will generate a pdf for you.
```
Comment (jvasseur) [inline README.md:29]:
```suggestion
You then will be able to make a `POST` request to `localhost:3000` with HTML data and the response will generate a PDF for you.
```

**Review by jvasseur — CHANGES_REQUESTED** [review]
Sorry for not catching everything on the first review

**Review by jvasseur — APPROVED** [review]

**Pull request merged**